docs/panel/advanced/docker.mdx

```diff
@@ -0,0 +1,85 @@
+
+- `APP_INSTALLED`: If you are manually configuring the panel via environment variables, set it to `true` to disable the installer.
+- `APP_KEY`: A 32-character-long randomly-generated string used as an encryption key.
+- `APP_TIMEZONE`: The timezone for your panel. See https://www.php.net/manual/en/timezones.php for a list of options.
+- `APP_NAME`: This will be the Name of your Panel. Default: Pelican
+- `DB_CONNECTION`: The type of database to use for the panel. `[sqlite, mariadb, mysql]`
+- `CACHE_STORE`: The driver used for caching. We recommend "Filesystem." `[file, redis]`
+- `SESSION_DRIVER`: The driver used for storing sessions. We recommend "Filesystem" or "Database." `[file, database, cookie, redis]`
+- `QUEUE_CONNECTION`: The driver used for handling queues. We recommend "Database." `[database, redis, sync]`
+
+:::warning
+    If you are using `redis` for your `SESSION_DRIVER` or `QUEUE_CONNECTION`, you must set `CACHE_STORE` to `redis`.
+:::
+
+#### If using SQLite:
+- `DB_DATABASE`: The path of your .sqlite file relative to the database folder.
+
+#### If using MySQL or MariaDB
+- `DB_HOST`: The host of your database.
+- `DB_PORT`: The port of your database.
+- `DB_DATABASE`: The name of the panel database.
+- `DB_USER`: The name of your database user.
+- `DB_PASSWORD`: The password of your database user.
+
+#### If using Redis
+- `REDIS_HOST`: The host of your redis server. Make sure it is reachable.
+- `REDIS_PORT`: The port of your redis server.
+- `REDIS_USERNAME`: **Optional** The name of your redis user.
+- `REDIS_PASSWORD`: **Optional** The password for your redis user.
+
+### Configured in Settings
+These options are not configured by the installer but can be set in the admin settings area, or using these environment variables.
+
+#### General
+- `APP_FAVICON`: A path relative to the public directory to use as the favicon. Default: `/pelican.ico`
+- `FILAMENT_TOP_NAVIGATION`: Set to `false` to use the sidebar for navigation. Set to `true` to use a top menu. `[true, false]` Default: false
+- `PANEL_USE_BINARY_PREFIX`: Whether the panel should use the binary prefix to display units (MiB/GiB) or not (MB/GB). `[true, false]` Default: true
+- `APP_2FA_REQUIRED`: Set to `0` to disable 2FA requirement. Set to `1` to enable for admins only. Set to `2` to enable for everyone. `[0, 1, 2]` Default: 0
+- `TRUSTED_PROXIES`: A comma-separated list of trusted proxies. `*` can be used to trust any proxy that connects directly to your server, a requirement when you cannot know the address of your proxy (e.g. if using Rackspace balancers). `**` can be used to trust not just any proxy that connects directly to your server, but also proxies that connect to those proxies, and all the way back until you reach the original source IP. Default: null
+- `FILAMENT_WIDTH`: The max display width. Default: `screen-2xl`
```

Comment:
I'm not sure where to find the options for FILAMENT_WIDTH (other than in the panel, which seems to be in a different format) Is there a list somewhere I can include as options?

Reply:
Its a filament enum